cm/cm.c

```diff
@@ -1128,8 +1133,10 @@ M0_INTERNAL int m0_cm_complete(struct m0_cm *cm)
 M0_INTERNAL void m0_cm_complete_notify(struct m0_cm *cm)
 {
 	M0_ASSERT(m0_cm_is_locked(cm));
+	M0_ENTRY("Notifying cm %p", cm);
```

Comment:
Currently the pointer of CM is more helpful, because "cm" is passed as pointer between function calls.
But you are right. We may need this id. So I will add this id in messages.